### DISCUSSION TITLE
Exclude Namespace

### DISCUSSION THREAD

**Discussion by shadowbreakerr** [aquasecurity/starboard#638] [Development]
Would it be possible to add an "excludeNamespaces" condition to the operator?

Currently when operating in all namespaces (targetNamespaces: "") I've noticed issues whereby the Rancher (2.5.7) agent is timing out whilst waiting for the operator scan to complete, which causes the API to temporarily timeout and subsequently any deployments etc. to fail.

It would be really useful to be able to exclude a namespace from the operator instead of having to update the ever growing list of "allowed" namespaces.

**Comment by danielpacak** [comment]
👋 @shadowbreakerr Thank you for your inputs. I think it's a reasonable requirement. Please follow the general discussion #670 where I included your use case.
